client/src/components/screens/PatientSubmission/components/SecondPage.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import { FormInstance } from 'antd';
+import Practitioners, { PractitionerData } from './Practitioners';
+import Api from '../../../../helpers/api';
+
+interface Props {
+  form: FormInstance
+  doctorOptions: {
+    optionSelected: (value: PractitionerData | null) => void
+    initialValue: string
+  },
+  residentOptions: {
+    optionSelected: (value: PractitionerData | null) => void
+    initialValue: string
+  }
+}
+
+async function searchPractitioner(term: string): Promise<PractitionerData[]> {
+  if (term == null) {
+    return [];
+  }
```

Comment:
- Check for the term's length here
- May be good to log the error before returning the empty array

Reply:
I moved this code from somewhere else but that if doesn't make sense in this context. I changed it for `!term` 'cause you don't want to search for an empty string. I didn't add a log because it's not an error to have an empty string (if the user deletes every thing in the field for example)